mangaki/mangaki/utils/anidb.py

```diff
@@ -93,7 +93,7 @@ def lang_map(self) -> Dict[str, ExtLanguage]:
 
     @cached_property
     def unknown_language(self) -> ExtLanguage:
-        return ExtLanguage.objects.get(source='anidb', ext_lang='x-unk')
+        return ExtLanguage.objects.filter(source='anidb', ext_lang='x-unk').first()
```

Comment:
What happened on your side to make this change?
Theorically, the couple `('anidb', 'x-unk')` should be unique in `ExtLanguage` :/.